packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -3,16 +3,15 @@ sidebar_position: 2
 title: <ErrorBoundary/>
 ---
 
-This component provide a simple and reusable wrapper that you can use to wrap around your components. Any rendering errors in your components hierarchy can then be gracefully handled.
+Any rendering errors in children of this component can be handled.
```

Comment:
Due to the `useErrorboundary`, errors other than rendering can be handled in the `Errorboundary`, is it okay not to specify that part?

Reply:
Wow... so good catch!👍